samples/samples/backup_sample.py

```diff
@@ -269,6 +274,45 @@ def update_backup(instance_id, backup_id):
 # [END spanner_update_backup]
 
 
+# [START spanner_create_database_with_version_retention_period]
+def create_database_with_version_retention_period(instance_id, database_id, retention_period):
+    """Creates a database with a version retention period."""
+    spanner_client = spanner.Client()
+    instance = spanner_client.instance(instance_id)
+    ddl_statements = [
+        "CREATE TABLE Singers ("
+        + "  SingerId   INT64 NOT NULL,"
+        + "  FirstName  STRING(1024),"
+        + "  LastName   STRING(1024),"
+        + "  SingerInfo BYTES(MAX)"
+        + ") PRIMARY KEY (SingerId)",
+        "CREATE TABLE Albums ("
+        + "  SingerId     INT64 NOT NULL,"
+        + "  AlbumId      INT64 NOT NULL,"
+        + "  AlbumTitle   STRING(MAX)"
+        + ") PRIMARY KEY (SingerId, AlbumId),"
+        + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE",
+        "ALTER DATABASE {}"
```

Comment:
As discussed, I think this will fix the `InvalidArgument` error:
```suggestion
        "ALTER DATABASE `{}`"
```